artmaker/src/main/java/io/artmaker/ArtMakerViewModel.kt

```diff
@@ -83,45 +87,30 @@ internal class ArtMakerViewModel(
     private val _finishedImage = MutableStateFlow<Bitmap?>(null)
     val finishedImage = _finishedImage.asStateFlow()
 
+    init {
+        listenToUndoRedoState()
+    }
+
     fun onAction(action: ArtMakerAction) {
         when (action) {
             is ArtMakerAction.TriggerArtExport -> triggerArtExport(action.type)
             is ArtMakerAction.ExportArt -> exportArt(action.bitmap)
-            ArtMakerAction.Redo -> redo()
```

Comment:
I removed these from the user actions because, in as much as they are user actions, they affect drawing. These are draw events now

Reply:
This is noted...